data/sql/updates/pending_db_world/rev_1625244599291561100.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1625244599291561100');
+
+UPDATE `quest_template` SET `QuestType` = 2 WHERE (`ID` = 1559);
```

Comment:
don't forget to add a new empty line at the end of file:

![image](https://user-images.githubusercontent.com/75517/124362897-4642e300-dc38-11eb-857d-96f3960bd68c.png)

Reply:
@FrancescoBorzi Thanks, already fixed it